### DISCUSSION TITLE
The formula returns the number of days from the given Date

### DISCUSSION THREAD

**Discussion by sergey-dme** [Budibase/budibase#1925] [Show and tell]
Hey all, Maybe it will be helpful in some use cases: 
The formula returns how many days ago a record has been created (Date/Time type field):

`{{ toFixed (divide (subtract (date now "x") (date Repeater.TableName.Created At "x")) 86400000) 0 }}`

Just replace `Repeater.TableName.Created At` to your Column address with Date/Time type.

**Comment by joebudi** [comment]
This is awesome @sergey-dme

**Reply by sergey-dme** [reply]
@joebudi  Thank to @mike12345567 Who initially led me to this, explaining how date timestamp in JS works here: https://github.com/Budibase/budibase/discussions/1910
